web-sdk/README.md

```diff
@@ -8,7 +8,7 @@
   alt="Preview of the web SDK"
 />
 
-After successfully tokenising and linking the card on Visa or MasterCard networks, Fidel API returns the created card object with a unique `id` that you must use to link each unique card and transaction to your user’s account. The `id` of each linked card is present on the transaction object as `cardId`. You can now create card-linked web and mobile applications with online and offline transactional data visibility in a matter of minutes.
+After successfully tokenising and linking the card on American Express, Visa or MasterCard networks, Fidel API returns the created card object with a unique `id` that you must use to link each unique card and transaction to your user’s account. The `id` of each linked card is present on the transaction object as `cardId`. You can now create card-linked web and mobile applications with online and offline transactional data visibility in a matter of minutes.
```

Comment:
We usually do reverse-alpha, "Visa, Mastercard or Amex", because it just rolls more smoothly. Please let's also fix the incorrect spelling of Mastercard (they don't capitalise the `c`)

Reply:
Ah-ha! at least the Typo wasnt on me- that was already there!